compiler/damlc/tests/src/DA/Test/DamlcIntegration.hs

```diff
@@ -321,6 +321,7 @@ mainProj TestArguments{..} service outdir log file = do
     runActionSync service $ do
             cores <- ghcCompile log file
             corePrettyPrint cores
+            getDlintIdeas file
```

Comment:
@shayne-fletcher-da If I move this two lines up, everything breaks with "bad dependency" errors. To me this looks like the Shake rule for the linter has an issue. Could you please have a look. (Or tell me I'm wrong and using things incorrectly.)

Reply:
Moving this after the typechecker defeats the whole purpose. Now you’re back to the lint rule potentially not running if typechecking fails.

Reply:
Yeah, you're right. I shouldn't code after midnight...

Reply:
Coding after midnight is rarely a good idea :wink:

Reply:
@cocreature pulled a fix out of his magic bag this morning in https://github.com/digital-asset/daml/pull/3045 :)